assets/js/modules/analytics/datastore/settings.test.js

```diff
@@ -333,6 +339,70 @@ describe( 'modules/analytics settings', () => {
 
 				expect( ( await getItem( cacheKey ) ).value ).toBeFalsy();
 			} );
+
+			describe( 'analytics-4', () => {
+				beforeEach( () => {
+					registry.dispatch( STORE_NAME ).receiveGetExistingTag( null );
+					registry.dispatch( STORE_NAME ).setSettings( validSettings );
+
+					enabledFeatures.add( 'ga4setup' );
+				} );
+
+				afterEach( () => {
+					enabledFeatures.delete( 'ga4setup' );
```

Comment:
This shouldn't be necessary as `enabledFeatures` is already cleared after each test.

Reply:
Updated.